default/scripting/species/SP_LEMBALALAM.focs.txt

```diff
@@ -16,7 +16,7 @@ Species
 
     defaultfocus = "FOCUS_RESEARCH"
     likes = ["FOCUS_RESEARCH" "SPARK_FOSSILS_SPECIAL" "SUCCULENT_BARNACLES_SPECIAL" "ANCIENT_RUINS_DEPLETED_SPECIAL"]
-    dislikes = ["SUPERCONDUCTOR_SPECIAL" "SPICE_SPECIAL" "PROBIOTIC_SPECIAL" "POSITRONIUM_SPECIAL" "MONOPOLE_SPECIAL" "MINERALS_SPECIAL"]
+    dislikes = ["SUPERCONDUCTOR_SPECIAL" "SPICE_SPECIAL" "PROBIOTIC_SPECIAL" "POSITRONIUM_SPECIAL" "MONOPOLE_SPECIAL" "MINERALS_SPECIAL" "SP_NIGHTSIDERS" ]
```

Comment:
I don't think disliking other species does anything at present...?

Reply:
maybe not, but looks nice in the pedia page - e.g. "Species that dislike: ...", and fits their stories, and will be in place if it ever gets an effect.

and agreed i'll work on the first two icons a little more especially the sleepers. and look into those building reference errors probably just forgot some include..

and i'll squash again and rebase on master when i'm done with those points.. adding to top entry as checkmark tasks

Reply:
done.

Reply:
and ... forgot to adjust checksums

Reply:
and can't help myself but keep adding next species .. so back to draft. sorry.